stories/form/form-select/props-table.tsx

```diff
@@ -110,6 +110,18 @@ const DATA: ApiTableSectionProps[] = [
                     "The custom function to perform a search when the user enters a value in the search input",
                 propTypes: ["(searchValue: string) => T[]"],
             },
+            {
+                name: "renderListItem",
+                description:
+                    "Function to customise the rendering of a list item",
+                propTypes: ["(option: T) => JSX.Element"],
+            },
+            {
+                name: "renderCustomSelectedOption",
+                description:
+                    "Function to customise the rendering of the selected option object",
```

Comment:
Update description to:
"Function to customise the rendering of a list item's selected state"